src/app/public/modules/list-view-grid/list-view-grid.component.ts

```diff
@@ -388,6 +439,30 @@ export class SkyListViewGridComponent
     return true;
   }
 
+  private selectedMapEqual(prev: ListSelectedModel, next: ListSelectedModel): boolean {
+    if (prev.selectedIdMap.size !== next.selectedIdMap.size) {
+      return false;
+    }
+
+    for (let i = 0; i < next.selectedIdMap.size; i++) {
+      const key = Array.from(next.selectedIdMap.keys())[i];
```

Comment:
I could be wrong, but it appears `Array.from` is not supported by IE 11.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from

Reply:
Doh, good catch. Maps are tricky and there's only a limited amount of ways to iterate through them. I restructured this a bit to get around using `Array.from`.